_sips/process-specification.md

```diff
@@ -216,16 +221,16 @@ The SIP is now a recommendation for the compiler team or any other individual or
 group of people to provide an implementation of the proposal, as a pull request
 to the Scala 3 compiler repository. There is no set timeline for this phase.
 
-An implementation will be reviewed by the compiler team, and once the
-implementation is deemed good enough, it can ship as an Experimental feature in
-the following practical release of the compiler.
-
 Often, proposals not only need to be implemented in the compiler, but also in
 several other tools (IDEs, syntax highlighters, code formatters, etc.). As soon
-as a proposal reaches the implementation stage, the Committee notifies the
+as a proposal reaches the implementation stage, the Chairperson notifies the
 impacted tools that they should start implementing support for it. A list of
 tools of the ecosystem is maintained in [this document][tooling ecosystem].
 
+An implementation will be reviewed by the compiler team, and once the
+implementation is deemed good enough, it can ship as an Experimental feature in
+the following practical release of the compiler. At that point, the SIP Manager posts a follow-up comment on the Pre-SIP discussion to invite the community to test the feature and provide feedback.
```

Comment:
nitpick, but "the following practical release" isn't clear. as in, I genuinely don't know what that means.